App.js

```diff
@@ -133,12 +133,15 @@ const App = () => {
     console.log('Firebase setup already complete.');
   }
 
-  const user = firebase.auth().currentUser;
   let startupScreen = 'Login';
-  if (user) {
-    startupScreen = 'Root';
-    const displayName = user.displayName;
-  }
+
+  firebase.auth().onAuthStateChanged((user) => {
+    //user is already logged in
+    if (user) {
+      const displayName = user.displayName;
```

Comment:
Is this used anywhere?

Reply:
referring to the displayName variable

Reply:
well it will be used as the welcomeName once we update that from being 'Shriya'. But I can delete it since rn it's unnecessary